Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -321,7 +328,14 @@ class AEPEdgeFunctionalTests: FunctionalTestBase {
         XCTAssertEqual("val", requestBody["events[0].data.testDataDictionary.key"] as? String)
         XCTAssertEqual("app", requestBody["xdm.implementationDetails.environment"] as? String)
         XCTAssertEqual("\(MobileCore.extensionVersion)+\(Edge.extensionVersion)", requestBody["xdm.implementationDetails.version"] as? String)
+
+        #if os(iOS)
         XCTAssertEqual("https://ns.adobe.com/experience/mobilesdk/ios", requestBody["xdm.implementationDetails.name"] as? String)
+        #endif
+
+        #if os(tvOS)
+        XCTAssertEqual("https://ns.adobe.com/experience/mobilesdk/tvos", requestBody["xdm.implementationDetails.name"] as? String)
```

Comment:
let's create a constant for this outside the tests and use it without the platform switch. 
```
XCTAssertEqual(BASE_NAMESPACE, requestBody["xdm.implementationDetails.name"] as? String)
```